src/parser/jobs/gnb/modules/LightningShot.tsx

```diff
@@ -0,0 +1,49 @@
+import {Trans} from '@lingui/react'
+import {ActionLink} from 'components/ui/DbLink'
+import ACTIONS from 'data/ACTIONS'
+import {Event, Events} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {EventHook} from 'parser/core/Dispatcher'
+import {filter} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {Data} from 'parser/core/modules/Data'
+import {SimpleStatistic, Statistics} from 'parser/core/modules/Statistics'
+import React from 'react'
+
+export class LightningShot extends Analyser {
+	static override handle = 'lightningshot'
+
+	@dependency data!: Data
+	@dependency statistics!: Statistics
+
+	private shotsFired: number = 0
+
+	private windHook?: EventHook<Events['action']>
```

Comment:
I think this is leftover from whatever you copied this from, cause it's not used here. Lets clean that out before we forget.